testdata/src/main/java/com/hedvig/app/testdata/feature/home/builders/HomeDataBuilder.kt

```diff
@@ -16,7 +16,8 @@ data class HomeDataBuilder(
         CommonClaimBuilder(title = "Trasig telefon").build(),
         CommonClaimBuilder(title = "Försenat bagage").build()
     ),
-    private val importantMessages: List<HomeQuery.ImportantMessage> = emptyList()
+    private val importantMessages: List<HomeQuery.ImportantMessage> = emptyList(),
+    private val upcomingRenewal: HomeQuery.UpcomingRenewal? = null
```

Comment:
I think we can simplify this to something like `renewalDate: LocalDate? = null` and then generate the entire `HomeQuery.UpcomingRenewal` if it is not null. That saves effort on the calling side.